.github/scripts/assign_reviewers.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+async function run(github, context, core, fs) {
+  try {
+    // Pick two reviewers from list
+    const numberOfReviewers = 2;
+    const repo = context.repo;
+
+    // Read CODEOWNERS
+    const codeownersContent = fs.readFileSync('.github/CODEOWNERS', 'utf8');
+    const lines = codeownersContent.split('\n');
+
+    // Search COMMITTERS
+    const placeholderLine = lines.find(line => line.startsWith('COMMITTERS_PLACEHOLDER'));
+    if (!placeholderLine) {
+      throw new Error("No COMMITTERS found in CODEOWNERS");
+    }
+
+    // Extract committers from placeholder line
+    const committers = placeholderLine.match(/@[\w-]+/g).map(u => u.substring(1));
```

Comment:
Weird parsing. If we don't use CODEOWNER native function, why not simply configure a JS array:

```js
const candidate = [
  "Xuanwo",
  "Ji-Xinyou",
  ...
];
```

Reply:
GitHub's UI natively supports CODEOWNERS to ensure we reference the correct GitHub IDs.

![image](https://github.com/apache/incubator-opendal/assets/5351546/17a8b760-f5ec-42b3-927e-7f413b2525e8)